bloom/commands/git/release.py

```diff
@@ -259,6 +259,9 @@ def execute_track(track, track_dict, release_inc, pretend=True, debug=False, fas
     else:
         archive_file = '{name}.tar.gz'.format(**settings)
     settings['archive_path'] = os.path.join(archive_dir_path, archive_file)
+    actions_env = os.environ.copy()
```

Comment:
This copy used to occur inside the loop below for each action. I consider creating only one copy of the environment an optimization that is minor bordering on trivial for such a small expected loop size.

But since child process can't affect the environment of its parent or subsequent child processes this should be harmless.